gramps/plugins/db/dbapi/dbapi.py

```diff
@@ -83,6 +82,21 @@ class DBAPI(DbGeneric):
     def _initialize(self, directory, username, password):
         raise NotImplementedError
 
+    def supports_json_access(self):
+        """
+        A DBAPI level method for testing if the
+        database support JSON access.
+        """
+        try:
+            self.dbapi.execute("SELECT json_data from person limit 1;")
```

Comment:
@dsblank are both checks required? Could you just rely on reading from metadata alone?

Reply:
Yes, you are correct. I'll fix. Thanks!